Orange/util.py

```diff
@@ -0,0 +1,68 @@
+"""Various small utilities that might be useful everywhere"""
+
+from functools import wraps
+import sys
+import abc
+import numpy as np
+
+
+def scale(values, min=0, max=1):
+    """Return values scaled to [min, max]"""
+    ptp = np.nanmax(values) - np.nanmin(values)
+    if ptp == 0:
+        return np.clip(values, min, max)
```

Comment:
Did you consider using (min+max) / 2? This would produce the same scaled value for all constant arrays.